app/controllers/errors_controller.rb

```diff
@@ -22,4 +23,16 @@ def too_many_requests
   def internal_server_error
     render "internal_server_error", status: :internal_server_error
   end
+
+  def team_not_found
```

Comment:
Should this live somewhere else, in a different controller (probably in the Users namespace)? My intention with `errors_controller` was for it to only handle the fallback/generic 404/422/429/500 scenarios. It feels like this functionality doesn't fit that model.

Reply:
Ya, it's not a perfect fit. I think it could fit well in `Users::SessionsController` (fwiw I also considered and discarded putting it into `PagesController`)

Reply:
Oh ya, this is why I didn't do it, and decided to 🥾🥫 it into `ErrorsController`

<img width="701" alt="image" src="https://github.com/nhsuk/manage-vaccinations-in-schools/assets/15608/ef339182-ff81-4845-b264-7e3d028abd85">

Reply:
Found a way. I moved this to `users/accounts_controller` ... it could also have gone to `users/sessions_controller` but that requires a separate `devise_scope` as the above error indicates, and I think the accounts controller is as good a place as sessions.